File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppDeleteActionType.java
package com.iterable.iterableapi;

public enum IterableInAppDeleteActionType {

    INBOX_SWIPE {
        @Override
        public String toString() {
            return "inbox-swipe";
        }
    },

    DELETE_BUTTON {
        @Override
        public String toString() {
            return "delete-button";
        }
    },

    OTHER {
        @Override
        public String toString() {
            return "other";
        }
    }



}

File: iterableapi/src/main/java/com/iterable/iterableapi/IterableInAppDeleteActionType.kt
package com.iterable.iterableapi

enum class IterableInAppDeleteActionType {

    INBOX_SWIPE {
        override fun toString(): String {
            return "inbox-swipe"
        }
    },

    DELETE_BUTTON {
        override fun toString(): String {
            return "delete-button"
        }
    },

    OTHER {
        override fun toString(): String {
            return "other"
        }
    }

}
